Firestore/Source/Local/FSTLocalStore.mm

```diff
@@ -258,36 +258,25 @@ - (void)setLastStreamToken:(nullable NSData *)streamToken {
 }
 
 - (MaybeDocumentMap)applyRemoteEvent:(const RemoteEvent &)remoteEvent {
+  const SnapshotVersion &lastRemoteVersion = _queryCache->GetLastRemoteSnapshotVersion();
+
   return self.persistence.run("Apply remote event", [&] {
     // TODO(gsoltis): move the sequence number into the reference delegate.
     ListenSequenceNumber sequenceNumber = self.persistence.currentSequenceNumber;
 
-    DocumentKeySet authoritativeUpdates;
     for (const auto &entry : remoteEvent.target_changes()) {
       TargetId targetID = entry.first;
       const TargetChange &change = entry.second;
 
       // Do not ref/unref unassigned targetIDs - it may lead to leaks.
```

Comment:
I think this comment was removed on other platforms?

Reply:
Removed. Thanks for catching!